pkg/controllers/consolidation/controller.go

```diff
@@ -611,7 +632,7 @@ func (c *Controller) statefulSetsReady(ctx context.Context) bool {
 			// unspecified defaults to 1
```

Comment:
Should the `List` failure on line 625 be `return false` instead?

Reply:
Yup, missed one